config/locales/links/es.yml

```diff
@@ -10,7 +10,6 @@ es:
     back_to_sp: Volver a %{sp}
     cancel: Cancelar
     cancel_account_creation: "< Cancelar la creación de cuenta"
-    cancel_idv: "< Cancelar la verificación de cuenta"
```

Comment:
Maybe silly, but do you want to use the same ["guillemet"](https://en.wikipedia.org/wiki/%E2%80%B9) (left angle quote: '‹') here, as in the English and French versions? It appears to be supported by Español.

Reply:
1) How do you guys know what a guillemet is?
2) Is which one to use (I agree they should probably all be the same) a question for the designers? It's not something I touched here, other than removing a translation that happened to have one.

Reply:
Guillemets for the guillemet god

I don't want to speak for Michael, but I have bought, read, and recommend this book: http://www.shadycharacters.co.uk/

Yeah, you could ask a designer, but I don't think it matters a whole whole lot. I prefer the fancy punctuation for fancy punctuation's sake.

Reply:
TIL that: "Guillemet is a diminutive of the French name Guillaume (equivalent to English William), after the French printer and punchcutter Guillaume Le Bé (1525–98)," and "Some languages derive their word for guillemets analogously: the Irish term is Liamóg, from Liam 'William' and a diminutive suffix." [The latter possibly being humorous to someone--linguists?]